README.md

```diff
@@ -101,4 +102,10 @@ Type: `boolean`. Default: `false`.
 Enable if you need to add the old `:focus` instead of `:focus-visible` selector
 to every `:hover`.
 
+### `splitRules`
+
+Type: `boolean`. Default: `false`.
```

Comment:
Should we make it `true` by default?

Reply:
Makes sense given the current support. I've updated the PR. I didn't want to change the original tests, so I just added `shouldSplit: false` to them.